CHANGELOG.rst

```diff
@@ -14,6 +14,12 @@ Change Log
 Unreleased
 ----------
 
+[9.0.1] - 2023-10-32
+--------------------
+Changed
+~~~~~~~
+* Fixed key error in merging event producer configs
```

Comment:
```suggestion
* Fixed key error in merging event producer configs. Previously, setting only one of `enabled` or `event_key_field` would result in the other being overwritten with None.
```

Reply:
Actually it would result in a KeyError. Will update.

Reply:
Ah, right.